letsencrypt/Dockerfile

```diff
@@ -18,8 +19,20 @@ RUN apk add --no-cache --update \
         libffi-dev \
         musl-dev \
         openssl-dev \
+    && pip3 install setuptools==57.5.0 \
+    && echo "cffi==1.14.6" >> /tmp/requirements.txt \
+    && echo "cryptography==3.4.8" >> /tmp/requirements.txt \
+    && echo "setuptools==57.5.0" >> /tmp/requirements.txt \
```

Comment:
Why is that explizit needed?

Reply:
setuptools passed the specified version no longer allows 2to3 as part of the wheel build command. This breaks the package that Transip depends on suds-jurko. 57.5.0 is the latest version without that "feature" https://setuptools.pypa.io/en/latest/history.html#v58-0-0

As far as pinning the version of cryptography along with the fix to correctly getting the ARCH type allows me to pull the prebuilt newest wheel available from the home assistant wheel repo. Without this we would need to introduce the Rust installer script as the version of rust on alpine is currently too old to build cryptography. 

Let me know if you have any follow up questions.

Reply:
Could be because the ARCH arg is missing above

Reply:
You can just get the cryptography library from our wheel repo

Reply:
that is exactly where it's coming from. Without pinning it to a version however one of the many dependencies can try and request a "newer" one and the one from the repo gets ignored. the intent here is to make sure we grab it from the repo.